server/php-queries.h

```diff
@@ -34,7 +38,7 @@ struct net_event_t {
       const char *error_message;
     };
     struct { // job_worker_answer
-      void *job_result_script_memory_ptr;
+      job_workers::SharedMemorySlice *job_result_script_memory_ptr;
```

Comment:
It's not script memory anymore. Let's remove it to `job_result_memory_slice_ptr`